tools/js-sdk-release-tools/src/xlc/codeUpdate/updateUserAgent.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiVersionType, SDKType } from "../../common/types";
+import { getSDKType } from "../../common/utils";
+import { logger } from "../../utils/logger";
+
+const fs = require('fs');
+const path = require('path');
+
+export function changeClientFile(packageFolderPath: string, packageVersion: string){
```

Comment:
this function only update user agent? is so:
```suggestion
export function updateUserAgent(packageFolderPath: string, packageVersion: string){
```